R/easier-pkg.R

```diff
@@ -4,8 +4,11 @@
 #'
 #'
 #' @importFrom stats cor
-#' @importFrom utils install.packages installed.packages
+#' @importFrom utils globalVariables install.packages installed.packages
+#' @importFrom rlang .data
 #'
 #' @name easier-pkg
```

Comment:
We should consider adding
@importFrom graphics abline lines
@importFrom utils View 
to the NAMESPACE - here the directives for it